monkey/monkey_island/cc/resources/aws_exporter.py

```diff
@@ -0,0 +1,607 @@
+import logging
+import uuid
+from datetime import datetime
+import boto3
+from botocore.exceptions import UnknownServiceError
+
+from cc.resources.exporter import Exporter
+from cc.services.config import ConfigService
+from cc.environment.environment import load_server_configuration_from_file
+from common.cloud.aws import AWS
+
+logger = logging.getLogger(__name__)
+
+AWS_CRED_CONFIG_KEYS = [['cnc', 'aws_config', 'aws_access_key_id'],
+                        ['cnc', 'aws_config', 'aws_secret_access_key'],
+                        ['cnc', 'aws_config', 'aws_account_id']]
+
+
+class AWSExporter(Exporter):
```

Comment:
Split this into 2 files:
* The core of the exporter - what's in charge of putting everything together, the interface functions
* All the prepare_finding functions